loleaflet/Makefile.am

```diff
@@ -177,6 +177,11 @@ NODE_MODULES_JS_DST = $(patsubst %.js,$(DIST_FOLDER)/%.js,$(NODE_MODULES_JS))
 LOLEAFLET_LIBS_JS_SRC = $(patsubst %.js,$(srcdir)/js/%.js,$(LOLEAFLET_LIBS_JS))
 LOLEAFLET_LIBS_JS_DST = $(patsubst %.js,$(DIST_FOLDER)/%.js,$(LOLEAFLET_LIBS_JS))
 
+CANVASSECTION:
+	$(srcdir)/node_modules/typescript/bin/tsc --project $(srcdir)/tsconfig.json
```

Comment:
One suggestion : Is it possible to write the js file directly to the DIST_FOLDER/<...>/ here via tsconfig.json ? so that we can avoid the 'copy' steps (assuming the js linter will not complain because of this).

Reply:
Thank you for the suggestion, i will try it asap :)